crates/vault-registry/src/lib.rs

```diff
@@ -1244,6 +1260,7 @@ impl<T: Config> Pallet<T> {
         let mut vault = Self::get_active_rich_vault_from_id(&vault_id)?;
         let banned_until = height + Self::punishment_delay();
         vault.ban_until(banned_until);
+        ext::staking::force_refund::<T>(&vault_id)?;
```

Comment:
Also not necessary, Vault is slashed in redeem proportionally and this may severely undercollateralize it.

Reply:
It was a requirement in the design doc that nominations are refunded upon liquidation and banning - we can reconsider these

Reply:
We cannot do this since it may undercollateralize the Vault - meaning automatic liquidation.

Reply:
We could do force refunding only if it doesn't cause undercollateralization in this case

Reply:
I think from a functional perspective it is far easier to reason about separate force withdrawals.